core/src/test/java/io/questdb/test/griffin/ExpressionParserFuzzTest.java

```diff
@@ -0,0 +1,424 @@
+/*******************************************************************************
```

Comment:
I guess the commented part from `ExpressionParserTest#testCorrectPrecedenceOfBasicOps()` should be uncommented by this PR.